spec/models/item_spec.rb

```diff
@@ -2,12 +2,18 @@
 
 RSpec.describe Item, type: :model do
 
-  describe 'validations' do
-    it "is valid with all fields" do
+  let!(:item) { FactoryGirl.create(:item) }
+  let(:item2) { FactoryGirl.create(:item, price: 3,
+                                          created_at: Date.today,
+                                          shelf_life_days: -1)
+                                          }
```

Comment:
`}` at 9, 42 is not aligned with `let(:item2) { FactoryGirl.create(:item, price: 3,` at 6, 2.